charts/rancher-backup/files/default-resourceset-contents/fleet.yaml

```diff
@@ -1,6 +1,14 @@
+- apiVersion: "management.cattle.io/v3"
+  kindsRegexp: "^fleetworkspaces$"
 - apiVersion: "v1"
   kindsRegexp: "^namespaces$"
   resourceNameRegexp: "^fleet-"
+  # also match any namespaces created for workspaces
+  labelSelectors:
```

Comment:
This actually modifies the existing rule to only select `namespace` resources that start with `fleet-` prefix and (as of this change) only those that have `app.kubernetes.io/managed-by=rancher` labels.

I suspect based on the comment this should be a new rule like:
```yaml
- apiVersion: "v1"
  kindsRegexp: "^namespaces$"
  labelSelectors:
    matchExpressions:
      - key: "app.kubernetes.io/managed-by"
        operator: "In"
        values: ["rancher"]
```

This way the existing rule is unchanged and similar to the comment we will "also match for ns created for workspaces"